api/app/src/external/commonwell/document/document-query.ts

```diff
@@ -209,8 +214,11 @@ async function downloadDocsAndUpsertFHIR({
       try {
         const primaryId = getDocumentPrimaryId(doc);
         const s3FileName = createS3FileName(patient.cxId, primaryId);
-        const s3File = await fileExists(s3FileName);
-        if (s3File) return;
+
+        if (!override) {
+          const s3File = await fileExists(s3FileName);
+          if (s3File) return;
+        }
```

Comment:
Can just skip this step as S3 upload overrides existing files - [source](https://docs.aws.amazon.com/AmazonS3/latest/userguide/upload-objects.html)